index.js

```diff
@@ -1,5 +1,4 @@
 "use strict";
-const { parse, compileScript, compileTemplate } = require('@vue/compiler-sfc');
 
 (function () {
   window.translateSFC = async function (source) {
```

Comment:
If possible, remove async/await keywords/func usages (introduced in #7). It seems that it is excessive here.